_studio/mfx_lib/cmrt_cross_platform/include/cmrt_cross_platform.h

```diff
@@ -69,9 +69,6 @@ class SurfaceIndex
 
 private:
     unsigned int index;
```

Comment:
Alexey, please, break this change into 3: 1) CMRT, 2) MCTF, 3) VPP. These are too far things to be combined together.